src/components/Location.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+// TODO: Location styling/icon that Siraj wants
+const LocationContainer = styled.button`
+  width: 10px;
+  height: 10px;
+  border-radius: 100%;
+  background: red;
+  transform: translate(-50%, -50%);
+`
+
+const Location = (props) => {
+  const { onClick, id } = props
+  return <LocationContainer onClick={() => onClick(id)} />
+}
+
+Location.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  id: PropTypes.number.isRequired,
```

Comment:
Same with "Cluster": does "Location" have to know its `id` for anything?

Reply:
Yes, we decided to pass the id to the location component because eventually we need to pass in the location id to get the location data